src/moonstone-samples/lib/CalendarSample.js

```diff
@@ -118,7 +118,7 @@ module.exports = kind({
 	setLocale: function(inSender, inEvent){
 		if (ilib) {
 			var locale = inEvent.selected.content,
-				val = (locale == 'Use Default Locale') ? null : locale;
+				val = (locale == 'Use Default Locale') ? '' : locale;
 			updateLocale(locale);
```

Comment:
I think this line is the issue. We're sanitizing the value above but passing the unsanitized value to `updateLocale()`. Any falsey value is acceptable to pass to `updateLocale()` because enyo-ilib will replace it with the result of `ilib.getLocale()` as @gouniLee suggests.